Synthesis failed for Google.Cloud.DocumentAI.V1Beta2

Hello! Autosynth couldn't regenerate Google.Cloud.DocumentAI.V1Beta2. :broken_heart:

Here's the output from running `synth.py`:

```
%at 3b775158caa20f218db9d5c1ed1f334e0f5e971d
2020-06-09 22:38:32,868 autosynth [DEBUG] > Running: git log -1 --pretty=%at c1fae183ddeef0c59538863eac611fd679d1b7fb
2020-06-09 22:38:32,871 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8dbe1d7f963adba47ea88205cae3ff6b97adb75e
2020-06-09 22:38:32,875 autosynth [DEBUG] > Running: git log -1 --pretty=%at e45a2f54dce1ac68497acf305145f9e5bb7e1e48
2020-06-09 22:38:32,878 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1db88f2a42f6b0911c26eaf83e77377252cf94cc
2020-06-09 22:38:32,882 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8f41c697c67d357665ce9afac18793c8f80ac73c
2020-06-09 22:38:32,885 autosynth [DEBUG] > Running: git log -1 --pretty=%at 51362f378a286494980ffc9a5199ab6e3cefd6d5
2020-06-09 22:38:32,888 autosynth [DEBUG] > Running: git log -1 --pretty=%at c77929eb8d6c7d6238f585cb9ee9d364893f9ed4
2020-06-09 22:38:32,892 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8070bcb140b06991a636d4ba7d665c5adab1675d
2020-06-09 22:38:32,895 autosynth [DEBUG] > Running: git log -1 --pretty=%at 86285bbd54fbf9708838219e3422aa47fb8fc0b0
2020-06-09 22:38:32,898 autosynth [DEBUG] > Running: git log -1 --pretty=%at ef180e07d360d86ce0f71c448edffbf6b7ceb8c9
2020-06-09 22:38:32,902 autosynth [DEBUG] > Running: git log -1 --pretty=%at 150cfa0d994f9656c57eb4ed50f49c2f2add93f2
2020-06-09 22:38:32,905 autosynth [DEBUG] > Running: git log -1 --pretty=%at 68c72c1d1ffff49b7d0019a21e65705b5d9c23c2
2020-06-09 22:38:32,908 autosynth [DEBUG] > Running: git log -1 --pretty=%at cd804bab06e46dd1a4f16c32155fd3cddb931b52
2020-06-09 22:38:32,911 autosynth [DEBUG] > Running: git log -1 --pretty=%at 40a354b0f47454be4986f9db504cca4d4371c8c3
2020-06-09 22:38:32,914 autosynth [DEBUG] > Running: git log -1 --pretty=%at dd244bb3a5023a4a9290b21dae6b99020c026123
2020-06-09 22:38:32,917 autosynth [DEBUG] > Running: git log -1 --pretty=%at 0fb2fbf04ff93836c4a712afa43d4df6a76d4643
2020-06-09 22:38:32,920 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3c5166c1db0f1804af7acdbbd79137630a00ce63
2020-06-09 22:38:32,924 autosynth [DEBUG] > Running: git log -1 --pretty=%at 148a3caa5f6735c49d57bb4c8001a276fcfad588
2020-06-09 22:38:32,927 autosynth [DEBUG] > Running: git log -1 --pretty=%at 60c7a4c6e8d1f4e4179d46bbad6239abc898dc40
2020-06-09 22:38:32,930 autosynth [DEBUG] > Running: git log -1 --pretty=%at ca1372c6d7bcb199638ebfdb40d2b2660bab7b88
2020-06-09 22:38:32,933 autosynth [DEBUG] > Running: git log -1 --pretty=%at ed1174659dc748903dbb635c7e01e38c3f812fbb
2020-06-09 22:38:33,103 autosynth [DEBUG] > Running: git checkout 38b26d539b477738f2720bc60470725aec6e07c6
Note: checking out '38b26d539b477738f2720bc60470725aec6e07c6'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 38b26d539 Release Google.Cloud.BigQuery.Connection.V1 version 1.0.0-beta01
2020-06-09 22:38:33,120 autosynth [DEBUG] > Running: git checkout ed1174659dc748903dbb635c7e01e38c3f812fbb
Note: checking out 'ed1174659dc748903dbb635c7e01e38c3f812fbb'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at ed117465 fix!: Provide C#, Ruby and PHP namespace/package options.
2020-06-09 22:38:33,132 autosynth [DEBUG] > Running: git branch -f autosynth-Google.Cloud.DocumentAI.V1Beta2-24
2020-06-09 22:38:33,137 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.DocumentAI.V1Beta2-24
Switched to branch 'autosynth-Google.Cloud.DocumentAI.V1Beta2-24'
2020-06-09 22:38:33,154 autosynth [INFO] > Running synthtool
2020-06-09 22:38:33,154 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-06-09 22:38:33,154 autosynth [DEBUG] > log_file_path: /tmpfs/src/github/synthtool/logs/googleapis/google-cloud-dotnet/apis/Google.Cloud.DocumentAI.V1Beta2/24/sponge_log.log
2020-06-09 22:38:33,156 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-06-09 22:38:33,377 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.DocumentAI.V1Beta2/synth.py.
On branch autosynth-Google.Cloud.DocumentAI.V1Beta2-24
nothing to commit, working tree clean
Skipping microgenerator fetch/build: already built, and running on Kokoro
Building existing version of Google.Cloud.DocumentAI.V1Beta2 for compatibility checking
Generating Google.Cloud.DocumentAI.V1Beta2
Running pre-generation script for Google.Cloud.DocumentAI.V1Beta2
google/cloud/documentai/v1beta2/geometry.proto:22:8: Option "csharp_namespace" was already set.
google/cloud/documentai/v1beta2/document.proto:20:1: Import "google/cloud/documentai/v1beta2/geometry.proto" was not found or had errors.
google/cloud/documentai/v1beta2/document.proto:164:7: "BoundingPoly" is not defined.
google/cloud/documentai/v1beta2/document.proto:394:5: "BoundingPoly" is not defined.
google/cloud/documentai/v1beta2/geometry.proto:22:8: Option "csharp_namespace" was already set.
google/cloud/documentai/v1beta2/document.proto:20:1: Import "google/cloud/documentai/v1beta2/geometry.proto" was not found or had errors.
google/cloud/documentai/v1beta2/document.proto:164:7: "BoundingPoly" is not defined.
google/cloud/documentai/v1beta2/document.proto:394:5: "BoundingPoly" is not defined.
rm: missing operand
Try 'rm --help' for more information.
2020-06-09 22:38:37,461 synthtool [ERROR] > Failed executing /bin/bash generateapis.sh --check_compatibility Google.Cloud.DocumentAI.V1Beta2:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-dotnet/apis/Google.Cloud.DocumentAI.V1Beta2/synth.py", line 26, in <module>
    hide_output = False)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '('/bin/bash', 'generateapis.sh', '--check_compatibility', 'Google.Cloud.DocumentAI.V1Beta2')' returned non-zero exit status 1.
2020-06-09 22:38:37,488 autosynth [ERROR] > Synthesis failed
2020-06-09 22:38:37,488 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 38b26d539 Release Google.Cloud.BigQuery.Connection.V1 version 1.0.0-beta01
2020-06-09 22:38:37,507 autosynth [DEBUG] > Running: git checkout autosynth-Google.Cloud.DocumentAI.V1Beta2
Switched to branch 'autosynth-Google.Cloud.DocumentAI.V1Beta2'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 619, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 599, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/aa06596a-90a1-42f7-8ba5-4aad22e68c3a/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-dotnet;failed=false).

--- a/apis/Google.Cloud.DocumentAI.V1Beta2/postgeneration.sh
+++ b/apis/Google.Cloud.DocumentAI.V1Beta2/postgeneration.sh
@@ -1,6 +1,0 @@
-#!/bin/bash
-
-set -e
-
-# Undo the changes made in pregeneration.sh
-git -C $GOOGLEAPIS checkout google/cloud/documentai/v1beta2

--- a/apis/Google.Cloud.DocumentAI.V1Beta2/pregeneration.sh
+++ b/apis/Google.Cloud.DocumentAI.V1Beta2/pregeneration.sh
@@ -1,8 +1,0 @@
-#!/bin/sh
-
-set -e
-
-# Temporary hack to add the C# namespace to the protos, until
-# they're present in GitHub.
-sed -i '/^option go_package/i option csharp_namespace = "Google.Cloud.DocumentAI.V1Beta2";' \
-  $GOOGLEAPIS/google/cloud/documentai/v1beta2/*.proto

